From 9f6e5d24ed8941b7a1e146cc6701ad57dc03355c Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Fri, 12 Apr 2024 08:42:30 +0200
Subject: [PATCH] com.webos.service.videooutput: Fix outbound permission error

Fixes:

Apr 12 00:04:29 qemux86-64 SettingsService[1385]: [] [pmlog] LS_REQUIRES_SECURITY {"CLIENT":"com.webos.service.videooutput","SERVICE":"com.webos.service.settings","CATEGORY":"/","METHOD":"getSystemSettings"} Service security groups don't allow method call.

Possibly contributes to YouTube that stopped working in recent releases.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

Upstream-Status: Pending [https://github.com/webosose/com.webos.service.videooutput/pull/2]
---
 files/sysbus/com.webos.service.videooutput.perm.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.videooutput.perm.json b/files/sysbus/com.webos.service.videooutput.perm.json
index 7d21236..4140e94 100755
--- a/files/sysbus/com.webos.service.videooutput.perm.json
+++ b/files/sysbus/com.webos.service.videooutput.perm.json
@@ -1,5 +1,5 @@
 {
   "com.webos.service.videooutput": [
-    "settings"
+    "settings.query"
   ]
 }
